cc/env/register.py

```diff
@@ -1,10 +1,18 @@
 from dataclasses import dataclass
-from typing import Callable
+from typing import Callable, List
 
 from dm_control import mujoco
 from dm_control.rl import control
 
-from .envs import two_segments_v1, two_segments_v2, two_segments_v3
+from .envs import two_segments
+from .envs.two_segments import CartParams, JointParams, SegmentTask
+
+
+def load_physics_wrapper(cart_params: List[CartParams]):
+    def load_physics():
```

Comment:
`_load_physics_two_segments` 
better indicate its private to module

also return type is missing